scripts/setup

```diff
@@ -11,17 +11,11 @@ set -e
 # Create REANA DB
 echo 'Creating REANA database...'
 invenio reana-db init
-invenio reana-users create_default info@reana.io  > /dev/null 2>&1
+invenio reana-users create_default info@reana.io
 echo 'REANA database created.'
 
 # Create Invenio DB
 echo 'Creating Invenio database...'
 invenio db init
 invenio db create
 echo 'Invenio database created.'
-
-# Create admin role to restrict access
-echo 'Creating admin role...'
-invenio roles create admin || echo "Admin role already exists"
-invenio access allow superuser-access role admin || echo "Admin role already has superuser-access"
-echo 'Admin role created.'
```

Comment:
Is it feasible to purify also some of the other verbose output bits we are getting as part of this same PR? Otherwise we can create an issue for later...
 
```console
$ kubectl logs server-659ff59456-klfsq server
...
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 250032 bytes (244 KB) for 4 cores
*** Operational MODE: preforking+threaded ***
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:835: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '

/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:835: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '

<unknown>:1: DeprecationWarning: invalid escape sequence \]

<unknown>:1: DeprecationWarning: invalid escape sequence \]

<unknown>:1: DeprecationWarning: invalid escape sequence \.

<unknown>:1: DeprecationWarning: invalid escape sequence \.

<unknown>:1: DeprecationWarning: invalid escape sequence \.

<unknown>:1: DeprecationWarning: invalid escape sequence \.

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \?

<unknown>:1: DeprecationWarning: invalid escape sequence \?

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \?

<unknown>:1: DeprecationWarning: invalid escape sequence \?

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \]

<unknown>:1: DeprecationWarning: invalid escape sequence \]

<unknown>:1: DeprecationWarning: invalid escape sequence \.

<unknown>:1: DeprecationWarning: invalid escape sequence \.

<unknown>:1: DeprecationWarning: invalid escape sequence \.

<unknown>:1: DeprecationWarning: invalid escape sequence \.

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \?

<unknown>:1: DeprecationWarning: invalid escape sequence \?

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \?

<unknown>:1: DeprecationWarning: invalid escape sequence \?

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

<unknown>:1: DeprecationWarning: invalid escape sequence \[

WSGI app 0 (mountpoint='') ready in 8 seconds on interpreter 0x562e60a42d60 pid: 29 (default app)
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 29)
spawned uWSGI worker 1 (pid: 35, cores: 2)
spawned uWSGI worker 2 (pid: 37, cores: 2)
```

Reply:
Seems that those deprecation warnings do not come from our application, and they show up only because `set -e` is at the beginning of the command to error out if there are problems. Since the [`./scripts/setup` already has its `set -e` inside](https://github.com/reanahub/reana-server/blob/ca50d793c94ae36e7c8434191fc457f375b6b62c/scripts/setup#L9), we do not need to include it in the Dockerfile command, PR updated :)